### PR TITLE
Added 'response' event to propagate GraphHopper credit update

### PR DESCRIPTION
**Pull request by opoto**
Also upgraded example to leaflet 1.0.3 (and its included dist/leaflet-routing-machine.js URL, please check this is ok)

### CODE REVIEW

File: src/L.Routing.GraphHopper.js [hunk 1]
```diff
@@ -54,6 +54,14 @@
 				var data;
 
 				clearTimeout(timer);
+				var fired = err ? err : resp;
```
Comment (perliedman) [inline src/L.Routing.GraphHopper.js]: I'm thinking this event should only fire if the request hasn't timed out, so move it inside the conditional below. As it is now, you will get `response` events also for timed out requests, which could be confusing, although it's really an edge case.

**Review by perliedman — CHANGES_REQUESTED** [review]
Looking good! Except for my comment, this looks good to merge!

**Comment by opoto** [comment]
Good point, I updated as suggested.

**Comment by opoto** [comment]
Any comment?

**Comment by perliedman** [comment]
:+1:

**Comment by perliedman** [comment]
Published a new version, 1.2.0, with this included.

**Pull request merged**